bluemira/base/logs.py

```diff
@@ -93,8 +105,12 @@ def set_log_level(
         logger = logging.getLogger(logger_name)
 
         current_level = logger.getEffectiveLevel() if increase else 0
-        new_level = _convert_log_level(verbose, current_level)
-        _modify_handler(new_level, logger)
+        _modify_handler(
+            LogLevel(verbose)
+            if isinstance(verbose, str)
+            else LogLevel(int(current_level + (verbose * 10))),
```

Comment:
I understand logging is very jank at the moment but I'm struggling to read the whole   log level *10 I've seen in the PR.

Esp. here with int(current_level + (verbose * 10))

If this kinda of math is needed for some logging stuff, it should be abstracted away into a function. Otherwise it's difficult to read and understand why/what's going on.

Reply:
Issue opened in #3399